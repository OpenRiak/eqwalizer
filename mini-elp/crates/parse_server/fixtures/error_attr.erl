%%% Copyright (c) Meta Platforms, Inc. and affiliates. All rights reserved.
%%%
%%% This source code is licensed under the Apache 2.0 license found in
%%% the LICENSE file in the root directory of this source tree.

-module(error_attr).

-error("alamakota").

-error("two", "terms").

-error(NotATerm).
