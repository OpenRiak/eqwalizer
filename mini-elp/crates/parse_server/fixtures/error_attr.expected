[
    ParseError {
        path: "fixtures/error_attr.erl",
        location: Some(
            TextRange(
                239..244,
            ),
        ),
        msg: "-error(\"alamakota\").",
    },
    ParseError {
        path: "fixtures/error_attr.erl",
        location: Some(
            TextRange(
                261..266,
            ),
        ),
        msg: "badly formed 'error'",
    },
    ParseError {
        path: "fixtures/error_attr.erl",
        location: Some(
            TextRange(
                286..291,
            ),
        ),
        msg: "badly formed 'error'",
    },
]
